### PR TITLE
Test button Fix

### LINKED ISSUE #310: Fix Test Button on Testnet
Test button not shown as active in testnet, and should link back to mainnet when clicked:

Image

### PR DESCRIPTION
**Pull request by Eren-Yeaager**
Resolves #310 

https://github.com/NEARBuilders/gateway/assets/91129111/35e80a84-5afc-480c-bc22-eda66dd302ab

https://github.com/NEARBuilders/gateway/assets/91129111/07348937-b711-4dd6-ae51-2fcb0d4c8e64

### CODE REVIEW

**Comment by Jikugodwill** [comment]
I think it should be pointed to testnet when we're on mainnet and back to mainnet when we're on testnet (the icon could also be changed on testnet to a production related icon like the contruction hat or something - ⚙ )

File: apps/old/widget/components/Navbar.jsx [hunk 1]
```diff
@@ -242,11 +245,15 @@ function Navbar(props) {
...
+
+    const isTestnet = networkId === "testnet";
+
     return (
       <a
         target="_blank"
-        className="grey-btn"
-        href="https://test.nearbuilders.org"
+        className={`grey-btn ${isTestnet ? "testnet" : ""}`}
+        href="https://www.nearbuilders.org/"
```
Comment (Jikugodwill) [inline apps/old/widget/components/Navbar.jsx:256]:
```suggestion
        href={isTestnet ? `https://www.nearbuilders.org/`:`https://test.nearbuilders.org`}
```

**Comment by Jikugodwill** [comment]
Now what's left is the icon changes

File: apps/old/widget/components/Navbar.jsx [hunk 2]
```diff
@@ -242,11 +245,18 @@ function Navbar(props) {
...
-        href="https://test.nearbuilders.org"
+        className={`grey-btn ${isTestnet ? "testnet" : ""}`}
+        href={
+          isTestnet
+            ? "https://www.nearbuilders.org/"
+            : "https://test.nearbuilders.org/"
+        }
       >
         <img
           src="https://ipfs.near.social/ipfs/bafkreieud33bpqibciatt6uwqju4r3xk7jwy3bunfgiz35oiwyiapbcjbq"
```
Comment (Jikugodwill) [inline apps/old/widget/components/Navbar.jsx:262]: You can use this icon `<i className="bi bi-wifi"></i>` and follow the new design form the figma file
 - [Figma](https://www.figma.com/design/O9VBPoKQH80YLndvcbj3Zf/[WorkSpace]-HomePage-2.0?node-id=0-1&t=Mkv1CWCN2WsaN7RN-0)
 - Image
 
Screen Shot 2024-05-21 at 4 12 49 PM

**Review by Jikugodwill — CHANGES_REQUESTED** [review]

**Review by Jikugodwill — COMMENTED** [review]
You can use this icon `<i className="bi bi-wifi"></i>` and follow the new design form the figma file

- [Figma](https://www.figma.com/design/O9VBPoKQH80YLndvcbj3Zf/%5BWorkSpace%5D-HomePage-2.0?node-id=0-1&t=Mkv1CWCN2WsaN7RN-0)

- Image

Screen Shot 2024-05-21 at 4 12 49 PM

**Review by Megha-Dev-19 — APPROVED** [review]
LGTM

**Comment by Jikugodwill** [comment]
How about the same reflection on the gateway?

**Pull request merged**